public/projects.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Anita Wang | Web Developer</title>
+    <meta name="description" content="Web developer in training at Ada Developers Academy"
+  </head>
+
+<body>
+  <header>
+    <h1>Anita Wang | Web Developer</h1>
+    <nav>
+        <a href="/">Home</a>  |  
+        <a href="/about">About</a>  |  
+        <a href="/blog">Blog</a>  |  
+        <a href="/projects">Projects</a>  |  
+    </nav>
+  </header>
+
+  <h3>Projects</h3>
```

Comment:
For semantics, I'd love to see this part of the code wrapped in a `<section>` or `<div>` tag. It's separate the header and footer; wrapping it would show that you've got a repeatable hierarchy of header, content, footer.